srcpkgs/mypaint/template

```diff
@@ -1,19 +1,29 @@
 # Template file for 'mypaint'
 pkgname=mypaint
-version=1.2.1
-revision=5
+version=1.2.2
+revision=1
+_sversion=beta.0
+_libcommit=f6278c97aa5af443e30069099c6461b056cdfe63
+archs="x86_64* i686*"
```

Comment:
This is wrong, problem is not architecture but crosscompiling. Appending ` numpy_include=${XBPS_CROSS_BASE}/usr/lib/python3.8/site-packages/numpy/core/include/` to scons commands fixes build. Using scons build-style should do too, please check.

Reply:
That seems to have worked. Thanks.